can/interfaces/seeedstudio/seeedstudio.py

```diff
@@ -262,7 +263,9 @@ def _recv_internal(self, timeout):
         return None, None
 
     def fileno(self):
-        if hasattr(self.ser, "fileno"):
+        try:
             return self.ser.fileno()
-        # Return an invalid file descriptor on Windows
+        except io.UnsupportedOperation:
+            raise NotImplementedError("fileno is not implemented using current CAN bus")
+
         return -1
```

Comment:
This line now looks unreachable, and should be removed.

Reply:
@hardbyte I've amended the commit to remote that line 👍